tests/unit/lib/connect-test.js

```diff
@@ -0,0 +1,48 @@
+import Ember from 'ember';
+import connect from 'ember-cli-redux/lib/connect';
+import { module, test } from 'qunit';
+
+module('Unit | Lib | Connect');
+
+function createFakeReduxService(state = {}, dispatch = null) {
+  return Ember.Object.extend({state, dispatch}).create();
+}
+
+test('it works', function(assert) {
+  const subject = connect();
+  assert.ok(subject);
+});
+
+test('statemap strings become one way aliases', function(assert) {
```

Comment:
P2: Suggest `string in statemap hash produces one way alias of reduxService property`.